### ISSUE TITLE
Anchor Tag Contrast (Accessibility)

### ISSUE THREAD

**Issue by darmentrout** [oxalorg/sakura#7]
The color contrast between link text (#2c8882) and the background (#f9f9f9) does not meet WCAG 2.0 AA or AAA standards. This is an accessibility issue for people with various vision impairments. Darkening the link color to #246f6a will satisfy WCAG AA. 

Great work on this! I told everyone I work with about it and they are jazzed.

**Comment by purag** [comment]
I think the dark theme links need more contrast as well.
